LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
Isn't there a license on main already? I'm confused as to why this is showing up as a new file in the PR.

Reply:
I think this is the same license ... I copied it locally to be able to upload to Conda Forge, which requires a license file. I think nothing will change when committing this.